devfiles/nodejs/build/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Install the app dependencies in a full Node docker image
+FROM registry.access.redhat.com/ubi8/nodejs-12:1-36
```

Comment:
should the tag here be "latest"?  so this doesn't get out of date, especially for CVE fixes

Reply:
The version was fixed to keep the application builds reproducible. I have updated it to `1-45`.

Reply:
I see that `1`, `1-45` and `latest` all resolve to the same image today. But this will probably change over time and will need manual updates, so using a tag that only allows security fixes, but no other changes might be best.

@lholmquist Do you know what the tag `1` is for?

Reply:
Ah ok. then i guess it makes sense to leave it as is.  The numbering has something to do with the build number.

Reply:
Great!